jenkins/stage-maven-release.jenkinsfile

```diff
@@ -0,0 +1,110 @@
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2'
+            args '-e JAVA_HOME=/opt/java/openjdk-11'
+            alwaysPull true
+        }
+    }
+    environment {
+        VERSION = "${params.VERSION}"
+        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/client/opensearch-java/${VERSION}"
+    }
+    stages {
+        stage('parameters') {
+            steps {
+                script {
+                    properties([
+                        parameters([
+                            string(
+                                name: 'REF',
+                                trim: true
+                            ),
+                            string(
+                                name: 'VERSION',
+                                trim: true
+                            )
+                        ])
+                    ])
+                    if (params.REF.isEmpty() || params.VERSION.isEmpty()) {
+                        currentBuild.result = 'ABORTED'
+                        error('Missing REF and/or VERSION.')
+                    }
+                }
+            }
+        }
+        stage('Publish to Maven Local') {
+            steps {
+                // checkout the commit
+                git url: 'https://github.com/opensearch-project/opensearch-java.git', branch: 'main'
+                sh('git checkout ${REF}')
+
+                // publish maven artifacts
+                sh('./gradlew --no-daemon publishPublishMavenPublicationToLocalRepoRepository')
+            }
+        }
+        stage('Sign') {
```

Comment:
Change this entire block to

        stage('Sign') {
            steps {
                script {
                    signArtifacts(
                            artifactPath: "${ARTIFACT_PATH}",
                            type: maven,
                            platform: 'linux'
                    )
                }
            }
        }

Similar to https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile#L141-L151